app/src/main/res/xml/data_choices_preferences.xml

```diff
@@ -4,19 +4,22 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
-    <androidx.preference.SwitchPreference
+    <SwitchPreference
         android:key="@string/pref_key_telemetry"
         android:summary="@string/preferences_usage_data_description"
         android:title="@string/preference_usage_data"
+        android:layout="@layout/checkbox_right_preference"
         app:iconSpaceReserved="false" />
```

Comment:
Setting `app:iconSpaceReserved="true"` (or removing it as true is the default) seems to set a 72dp margin, so a custom layout isn't needed.

Reply:
Nice :)
(currently swearing to himself)